ivy_tests/test_ivy/test_frontends/test_paddle/test_fft.py

```diff
@@ -242,3 +242,39 @@ def test_paddle_rfftfreq(
         n=n,
         d=d,
     )
+
+
+@handle_frontend_test(
+    fn_tree="paddle.fft.irfftn",
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("complex"),
+        min_value=-10,
+        max_value=10,
+        min_num_dims=1,
+        max_num_dims=4,
+        min_dim_size=2,
+        valid_axis=True,
+        force_int_axis=True,
+    ),
+    norm=st.sampled_from(["backward"]),
+)
+def test_paddle_irfftn(
+    dtype_x_axis,
+    norm,
+    frontend,
+    test_flags,
+    fn_tree,
+    backend_fw,
+):
+    input_dtypes, x, axis = dtype_x_axis
+    helpers.test_frontend_function(
+        input_dtypes=input_dtypes,
+        backend_to_test=backend_fw,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        x=x[0],
+        s=None,
+        axes=[-1],
```

Comment:
As I mentioned on the other PR, need to test axes=axis rather than just -1 here.

Reply:
hello i have test, the axes can be 0 or 1 or -1 just in the range of the dim of axes, but the frontend is always give 0 in case,
and frontend doesn't support a list of axes to be input, so here it doesn't matter its value for test purpose